include/deal.II/distributed/tria.h

```diff
@@ -406,6 +406,15 @@ namespace parallel
                            const std::vector<CellData<dim>> &  cells,
                            const SubCellData &subcelldata) override;
 
+      /*
+       * @copydoc Triangulation::create_triangulation()
+       *
+       * @note Not inmplemented yet.
+       */
+      virtual void
+      create_triangulation(const construction::ConstructionData<dim, spacedim>
+                             &construction_data) override;
```

Comment:
This function shouldn't be very hard to implement, though. You'd just need to call the corresponding function of the base class and then just duplicate the logic of what happens after construction from the other `create_triangulation()` function of this class.

Reply:
Future work...